jekyll/_layouts/classic-docs.html

```diff
@@ -35,14 +35,26 @@
             {% if page.page-type != "index" and page.page-type != "homepage" %}
             <div id="tooltip" role="tooltip" class="tooltip-popover"></div>
             <div id="tooltip-time" role="tooltip" class="tooltip-popover">
-              Last updated
+              Last updated • Reading time
               <div id='arrow' data-popper-arrow></div>
             </div>
-
             <div class="quick-info-list">
               <time aria-describedby="tooltip-time" id="time-posted-on" datetime="{{  page.last_modified_at | date_to_xmlschema }}">
                 <img src="{{ site.baseurl }}/assets/img/docs/duration.svg">
                 {% if page.last_modified_at %}<span id="time-posted-ago"></span>{% else %} {{ page.date | date: '%d-%b-%y' }} {% endif %}
+                <span class="ert" title="Estimated reading time">
+                  {% assign words_total = page.content | replace: '<script type="math/tex">', '' | replace: '<script type="math/tex; mode=display">', '' | replace: '</script>', '' | strip_html | number_of_words %}
```

Comment:
I don't know if I follow the logic. where is this LaTeX script coming from?

Reply:
It was a suggestion from the article I was reading to add it in. I thought that if we end up having it later down the road its already accounted for. I can remove it no worries!